addons/sale/models/sale.py

```diff
@@ -157,7 +163,8 @@ def _compute_tax_id(self):
     amount_untaxed = fields.Monetary(string='Untaxed Amount', store=True, readonly=True, compute='_amount_all', track_visibility='onchange')
     amount_tax = fields.Monetary(string='Taxes', store=True, readonly=True, compute='_amount_all')
     amount_total = fields.Monetary(string='Total', store=True, readonly=True, compute='_amount_all', track_visibility='always')
-
+    currency_rate = fields.Float(digits=(12, 6), default=1.0, help='The rate of the currency to the currency of rate 1 applicable at the date of the order', store=True, readonly=True, compute='_compute_rate')
```

Comment:
Not important: order the kwargs; string, compute, store, digits, readonly and help.
Not sur sure the default value is important as this is a computed field.